consumer/src/tree.rs

```diff
@@ -258,6 +258,45 @@ impl State {
     pub fn focus(&self) -> Option<Node<'_>> {
         self.focus_id().map(|id| self.node_by_id(id).unwrap())
     }
+
+    pub fn app_name(&self) -> Option<String> {
+        self.data.app_name.clone()
+    }
+
+    pub fn toolkit_name(&self) -> Option<String> {
+        self.data.toolkit_name.clone()
+    }
+
+    pub fn toolkit_version(&self) -> Option<String> {
+        self.data.toolkit_version.clone()
+    }
+
+    pub fn app_and_toolkit_description(&self) -> Option<String> {
```

Comment:
This function seem too Windows specific to stay here at the moment. If another adapter end up needing it then we can put it inside the consumer crate but I'd suggest moving it into `accesskit_windows` for now.